netman/api/doc_config/api_samples/get_switch_hostname_interfaces.json

```diff
@@ -33,6 +36,7 @@
       "port_mode": "bond_member",
       "access_vlan": null,
       "trunk_native_vlan": null,
-      "trunk_vlans": []
+      "trunk_vlans": [],
+      "auto_negotiation": false
```

Comment:
After explanation, we keep it this way for sake of uniformity. This will be revisited if we decide to change the naming convention at a later time.